content/docs/1.0.0/tutorial/deploy-your-microservice.md

```diff
@@ -25,53 +25,52 @@ You should see two, the first being [Docker Hub](https://hub.docker.com/), but w
   "certificate": "",
   "requiresCert": false,
   "username": "",
-  "password": "",
   "userEmail": "",
   "userId": null
 }
 ```
 
 The unique ID for the built-in registry is always `2`.
 
-Now that we have that registry ID, we can use it to add our Docker image to its catalog. We'll provide `yourname/moving-average:v1` as the x86-image of our microservice:
+Now that we have that registry ID, we can use it to add our Docker image to its catalog. We'll provide `iofog-tutorial/moving-average:v1` as the x86-image of our microservice:
 
-```sh
-# registry ID 2 is the internal private one provided by ioFog
-iofog-controller catalog add \
-  --name "Moving Average" \
-  --x86-image yourname/moving-average:v1 \
-  --registry-id 2 \
-  --user-id 1
+```bash
+$ docker exec -it iofog-controller \
+    iofog-controller catalog add \
+    --name "Moving Average" \
+    --x86-image lkrcal/moving-average:v1 \
+    --registry-id 2 \
+    --user-id 1
 ```
 
-This command will return a catalog ID that we'll use in the next step.
+This command will return a catalog ID that we'll use in the next step, in this case it is `105`.
 
 ## Add Your Microservice
 
 Now that the Docker image containing our microservice code is registered, we can spin up new copies of it also using the Controller.
 
 Instantiating a new microservice is done using the `microservice add` command. We need to provide several options, the most notable being the catalog ID we received in the previous section as well as a node UUID—which is the UUID of the edge node we want this microservice to run on.
```

Comment:
We need to provide several options **and** the most notable **is** the catalog ID we received in the previous section as well as a node UUID**,** which is the UUID of the edge node **that* this microservice **should** run on.